app/components/MeetingRoomFAQ.tsx

```diff
@@ -0,0 +1,24 @@
+import { Accordion } from "@trussworks/react-uswds";
+import type { AccordionItemProps } from "node_modules/@trussworks/react-uswds/lib/components/Accordion/Accordion";
+import { FAQ_DATA } from "~/shared/constants";
+
+function MeetingRoomFAQ() {
+    const accordionItems: AccordionItemProps[] = FAQ_DATA.map(
+        (entry, index): AccordionItemProps => ({
+            title: `${index + 1}. ${entry.question}`,
+            content: entry.answer,
+            expanded: false,
+            id: index.toString(),
+            headingLevel: "h4",
+        }),
+    );
+
+    return (
+        <div>
```

Comment:
Is an element necessary here? And if so, should it be a `div` or something else?

Reply:
Ah, yeah this can be an empty fragment. Thanks!